bigquery/README.md

```diff
@@ -10,6 +10,8 @@ create it by running `gcloud auth application-default login`.
 * Determine the project that will be billed for querying the BigQuery tables.
 Your account must have `bigquery.jobs.create` permission on this project; this
 includes any project where you have the Viewer/Editor/Owner role.
+* If you have not already, create the `data-explorer_default` docker network
```

Comment:
How about:

* If `docker network ls` doesn't show `data-explorer_default`, run `docker network create data-explorer_default` 

Most people won't need to manually create the network, because most people will have done the Quickstart in the other repo before this.

Reply:
Done.